src/polytopes.jl

```diff
@@ -193,7 +193,7 @@ function bridge(p::Polygon{Dim,T}; width=zero(T)) where {Dim,T}
   end
 
   # retrieve chains as vectors of coordinates
-  pchains = [coordinates.(vertices(c)) for c in chains(p)]
+  pchains = [coordinates.(parent(vertices(c))) for c in chains(p)]
```

Comment:
```suggestion
  pchains = [coordinates.(vertices(open!(c))) for c in chains(p)]
```
What about this? I think the intention becomes more clear? I find `parent` a bit hacky because it has nothing to do with the algorithm itself. What I really wanted in this line was the list of coordinates of the corresponding open chain.

I am assuming that calling `open!(c)` won't modify the original set of `chains(p)`. If it does, then we probably want to add `open(c) = copy(c) |> open!` first in a separate PR and then rebase this work on top of it. What do you think?

Reply:
Looks better indeed! It looks like we are allocating a new array in `chains(p)`, so this modification should not have side effects on the polygon.

Reply:
I've added `open/close` to the master branch in any case. I will accept the suggestion here and see if tests pass 👌🏽